routes/users.js

```diff
@@ -301,6 +299,4 @@ router.post("/register", async (req, res) => {
       });
     }
   }
-});
-
-module.exports = router;
```

Comment:
is that necessary to delete this module.exports?

Reply:
I just fixed it